src/tests/healthcareProfessionalQuery.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Can we fix these instead of commenting them out? 
We should either remove console logs or change the rule. 
For `any` we should either loosen the rules where they are ok in certain sections or start properly using the types. (Kinda the whole point we're using typescript right?)

Reply:
On test files I don't care about strong typing

Reply:
that's a good question 🤔 I'm guessing there's a way to scope rules, but never looked into it.

In my personal projects, I usually make the tests `.js` files instead so I don't have to deal with types.

Reply:
upon a thorough 2 min investigation, I found we can do [nested configs](- I noticed we aren't `await` ing them yet. (not just the new ones), so we should probably update). So we could make a separate, more relaxed eslint config for the testing folder

Reply:
Oh nice! I'lll look into that, especially for the console logs, which are often useful in tests